### PR TITLE
Account for additionalFields object props not existing as fields in BQ table

### PR DESCRIPTION
**Pull request by stulees**
## Description
Configure log params to take all `additionalFields` param object props and merge them to one value for inserting to `event_details` in the `Display_Events.events` BQ table with the exception of `storage` field. 

When `storage` value is present in `additionalFields` param, it is directly configured as value to `storage` field for inserting to `Display_Events.events` BQ table

## Motivation and Context
Component log calls are being changed to provide object containing `errorCode` property as the primary value for `event_details` field. What was previously configured for `event_details` in component log calls is now provided as `additionalFields` to process. In some situations, properties in `additionalFields` did not exist in the `Display_Events.events` BQ table and thus the insert failed. 

## How Has This Been Tested?
Tested with presentation https://apps.risevision.com/templates/edit/1bb41942-a8e5-45c6-b9b7-c79edfc6b6a3/?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f using Charles to map to local version of common-template and rise-data-counter. 

Forced format error on rise-data-counter directly in HTML when running in player and in shared schedule. Validated logs correctly inserted in both `Display_Events.events` table and `endpoint-event-logs.logs.eventLog` table. 

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a

### CODE REVIEW

**Comment by stulees** [comment]
@olegrise Please review, thanks

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
       - checkout
       - restore_cache:
           key: node-cache-{{ checksum "package.json" }}
+      - run: node --version
```
Comment (olegrise) [inline .circleci/config.yml:24]: @stulees do you want to keep this change?

Comment (stulees) [reply .circleci/config.yml:24]: @olegrise I was thinking it wouldn't hurt ...

**Review by olegrise — APPROVED** [review]
Looks good 👍

**Pull request merged**